Examples/Io/Performance/ActsExamples/Io/Performance/VertexPerformanceWriter.cpp

```diff
@@ -291,51 +306,53 @@ ActsExamples::ProcessCode ActsExamples::VertexPerformanceWriter::writeT(
         }
 
         const auto& majorityParticle = *it;
-        associatedTruthParticles.emplace_hint(associatedTruthParticles.end(),
-                                              majorityParticle);
+        trackParameters.push_back(trajectories.trackParameters(tip));
+        associatedTruthParticles.push_back(majorityParticle);
```

Comment:
I'm confused by line 309. If At line 247 you create a vector from the SimParticle Set, why are you pushing back here? Aren't you adding the particles to an already filled vector? 
We talked about this, was it checked?

Reply:
this is misleading in the github diff view. there is an `} else {` in-between